internal/service/domain/service_test.go

```diff
@@ -0,0 +1,46 @@
+package domain
+
+import (
+	"testing"
+
+	"github.com/green-ecolution/green-ecolution-backend/config"
+	"github.com/green-ecolution/green-ecolution-backend/internal/service"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage"
+	storageMock "github.com/green-ecolution/green-ecolution-backend/internal/storage/_mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewService(t *testing.T) {
```

Comment:
At this point I'm not familiar enough with go. But shouldn't it say `t.Run("should return xy", func(t *testing.T)` here like in the other tests?